pam/internal/adapter/newpasswordmodel.go

```diff
@@ -146,3 +151,21 @@ func (m newPasswordModel) Blur() {
 	}
 	m.focusableModels[m.focusIndex].Blur()
 }
+
+func (m *newPasswordModel) Clear() {
+	for _, fm := range m.focusableModels {
+		switch entry := fm.(type) {
+		case *textinputModel:
+			entry.SetValue("")
+		}
+	}
+	m.focusIndex = 0
+
+	for i, fm := range m.focusableModels {
+		if i != m.focusIndex {
+			fm.Blur()
+			continue
+		}
+		fm.Focus()
+	}
+}
```

Comment:
I'll move these changes to a separate commit when rebasing, don't worry!

Reply:
:+1: